### PR TITLE
[#72] [UI] As the user, I can see the survey question

### LINKED ISSUE #72: [UI] As the user, I can see the survey question
## Why

We need to see the questions of the survey.
 
## AC

- Show the counter
- Show close button
- Show the next button
- Show the questions

Simulator Screen Shot - iPhone 14 Pro - 2023-08-31 at 10 29 18

### PR DESCRIPTION
**Pull request by nkhanh44**
- Close #72 

## What happened 👀

- Implemented survey question screen
- Rename `survey_screen` -> `survey_detail_screen`, so I can get the survey_screen for the questions 🥲.

## Insight 📝

- I renamed the `survey_screen` to `survey_detail_screen`, but it still showed the changes, so no need to review it `survey_detail_screen` again 🙏 .

- The structure of the survey screen will contain the `survey_question`, and all `survey_answer` like thumbs-up - textfield... will be in the `survey_question`. 

👇 
```
survey_screen
--survey_question
----survey_answer
```

## Proof Of Work 📹

https://github.com/nimblehq/flutter-ic-khanh-thieu/assets/25881847/45d1be96-9124-4864-ad36-b2de9bfcb66f

### CODE REVIEW

File: lib/screens/survey/survey_screen.dart [hunk 1]
```diff
@@ -1,111 +1,132 @@
...
+    );
+  }
 
-  Widget _buildTitle(BuildContext context) {
-    return Text(
-      survey.title,
-      style: context.textTheme.titleMedium,
-      maxLines: 2,
+  Widget _buildBottomButtons(
+      {required bool isNext, required BuildContext context}) {
```
Comment (doannimble) [inline lib/screens/survey/survey_screen.dart:81]: Missing trailing commas

**Review by doannimble — COMMENTED** [review]
rest lgtm

File: assets/images/2.0x/close_button.png
[the remark this answers is not in the document]
Comment (nkhanh44) [reply assets/images/2.0x/close_button.png:1]: Thank you, that's a good suggestion 👍. However, for consistency with previous images that were using PNG, for now, I will keep using PNG. Is it ok? 🙏

File: lib/screens/survey/survey_screen.dart (hunk 1 above)
Comment (nkhanh44) [reply lib/screens/survey/survey_screen.dart]: Thanks 🙌, I updated all at  https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/73/commits/ff10effd50a4c943f645a2b9e6870fa984d98344

File: assets/images/2.0x/close_button.png
[the remark this answers is not in the document]
Comment (doannimble) [reply assets/images/2.0x/close_button.png:1]: @nkhanh44 I'm afraid that we should not keep it. It's better to update gradually. We will update the others later. Please keep in mind that we only use svg for simple icon like this 🙏

[the remark this answers is not in the document]
Comment (nkhanh44) [reply assets/images/2.0x/close_button.png:1]: @doannimble I updated close button using SVG at https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/73/commits/788902234c6e8255ce01875aa3cde815eeffdd33, Thank you 🙌

**Review by doannimble — APPROVED** [review]
Lgtm now 🚀

**Review by Thieurom — APPROVED** [review]
🚀

**Pull request merged**